script/test/cmd/fix_detached_head.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
```

Comment:
Would be good to leave a little note where this is coming from :) Source.

Also, missing license!